CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# Changelog
+
+## Version 1.0.1
+
+- Updated .NET Core references to 1.0.1.
```

Comment:
That's true, I forgot to add the new stuff, I was just focused on reconstructing the history, one sec...